src/core/mac/sub_mac.hpp

```diff
@@ -305,12 +305,15 @@ class SubMac : public InstanceLocator
      * started, `mState` will become `kStateCslSample`. But it could be doing `Sleep` or `Receive` at this moment
      * (depending on `mCslState`).
      *
+     * @param[in]  aPanChannel  The current phy channel used by the device. This param is passed for the case when CSL
```

Comment:
the comment seems to me not accurate - @p aPanChannel is always passed, but only take effect when CslChannel is not specified.